### PR TITLE
Update add-staff.md

### PR DESCRIPTION
**Pull request by msbemba**
added extra cmdlets for -AutoMapping:$false

Fixes #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10282

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/bookings/add-staff.md [hunk 1]
```diff
@@ -80,6 +80,12 @@ You may want to add a person to your staff list in Bookings without making them
     ```powershell
     Add-MailboxPermission -Identity <bookingmailbox@emailaddress> -User <adminusers@emailaddress> -AccessRights FullAccess -Deny:$false
     ```
+    
+    By default, the mailbox gets automatically mapped to outlook. If you donot want the booking mailbox to show in users outlook, assign full access with the following commands:
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/add-staff.md]:
```suggestion
    By default, the mailbox gets automatically mapped to Outlook. If you do not want the booking mailbox to appear in the user's Outlook, assign full access with the following commands:
```

File: microsoft-365/bookings/add-staff.md [hunk 2]
```diff
@@ -94,7 +100,12 @@ Here's an example PowerShell command to add Allie Bellew to the Contoso daycare
     ```powershell
     Add-MailboxPermission -Identity "daycare@contoso.com" -User "Allie Bellew" -AccessRights FullAccess -InheritanceType All
     ```
-
+     By default, the mailbox gets automatically mapped to outlook. If you donot want the booking mailbox to show in users outlook, assign full access with the following commands:
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/add-staff.md]:
```suggestion
     By default, the mailbox gets automatically mapped to Outlook. If you do not want the booking mailbox to appear in the user's Outlook, assign full access with the following commands:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

File: microsoft-365/bookings/add-staff.md [hunk 3]
```diff
@@ -94,7 +100,12 @@ Here's an example PowerShell command to add Allie Bellew to the Contoso daycare
     ```powershell
     Add-MailboxPermission -Identity "daycare@contoso.com" -User "Allie Bellew" -AccessRights FullAccess -InheritanceType All
     ```
-
+     By default, the mailbox gets automatically mapped to Outlook. If you do not want the booking mailbox to appear in the user's Outlook, assign full access with the following commands:
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/add-staff.md:103]:
```suggestion
     By default, the mailbox gets automatically mapped to Outlook. If you do not want the booking mailbox to appear in the user's Outlook, assign full access with the following commands:
```

Comment (msbemba) [inline microsoft-365/bookings/add-staff.md:103]: @JohanFreelancer9 I don't see the option to Commit

**Pull request merged**